src/Jackalope/ObjectManager.php

```diff
@@ -818,12 +818,12 @@ public function save()
             $this->executeOperations($this->operationsLog);
 
             // loop through cached nodes and commit all dirty and set them to clean.
-            if (isset($this->objectsByPath['Node'])) {
-                foreach ($this->objectsByPath['Node'] as $node) {
+            foreach ($this->objectsByPath as $nodes) {
```

Comment:
Ah, ok... but I need to save the version nodes... Maybe there should be a second option for workspace writes? (since somebody has to set the correct values on the version nodes...)